app/src/org/odk/collect/android/views/ResizingImageView.java

```diff
@@ -167,8 +167,7 @@ public void setFullScreen(){
             height = (int) (width / ratio);
         }
 
-        return new Pair<Integer, Integer>(new Double(width * imageScaleFactor).intValue(),
-                new Double(height * imageScaleFactor).intValue());
+        return new Pair<Integer, Integer>(new Double(width).intValue(), new Double(height).intValue());
```

Comment:
Hey, a little confused on what the ultimate goal of all the type conversions going on here is, but seems like between this line and the line above, something could definitely be simplified.